### DISCUSSION TITLE
Is it possible to drop SensioFrameworkExtraBundle in 5.4.x or only at 6.2.x?

### DISCUSSION THREAD

**Discussion by arderyp** [symfony/symfony#54116] [Q&A]
I am on the latest 5.4.x. I just migrated from annotations to attributes.  Everything is working great. grep is returning no instances of `Sensio` or `sensio` anywhere in my code. I dropped by sensio_framework_extra_bundle config. Everything is working.

And yet... when I remove the SensioFrameworkExtraBundle entry from `config/bundles.php`, my app breaks many routes with errors about being unable to map entity arguments in controller route parameters ([like this](https://stackoverflow.com/questions/66657307/cannot-autowire-argument-article-symfony-5)). 

It looks like `ParamConverterListener` is the source of my problem.  Does Symfony not have a native replacement for that until 6.2?

**Comment by MatTheCat** [comment]
The `EntityValueResolver` came with Symfony 6.2 so I'm not sure how you could be using attributes without Sensio's namespace 🤔 

See https://symfony.com/blog/new-in-symfony-6-2-built-in-cache-security-template-and-doctrine-attributes; you'd have to upgrade Symfony to get rid of the SensioFrameworkExtraBundle.

**Reply by arderyp** [reply]
Thanks for the comment.  Now, I am doubly confused lol, because attributes work fine.
